src/helpers/parse_snak.ts

```diff
@@ -105,20 +108,19 @@ for (const [ datatype, parser ] of Object.entries(parsers)) {
   normalizedParsers[normalizeDatatype(datatype)] = parser
 }
 
-export function parseClaim (datatype, datavalue, options, claimId) {
+export function parseSnak (datatype: DataType | void, datavalue: SnakValue, options: SimplifySnakOptions) {
   // Known case of missing datatype: form.claims, sense.claims, mediainfo.statements
-  datatype = datatype || datavalue.type
+  datatype = (datatype || datavalue.type) as DataType
```

Comment:
I wouldn't use `as` here and rather use `// @ts-expect-error` as the `as` hides the error

Reply:
I get `Unused '@ts-expect-error' directive`. Would that then be preferable to just remove the `as`, without `// @ts-expect-error` either?

Reply:
done in 0056c1b